i18n/hu.json

```diff
@@ -1,18 +1,18 @@
 {
     "_.code": "hu",
-    "_.name": "Magyar (hu)",
+    "_.name": "magyar (hu)",
```

Comment:
I checked the Wikipedia entry and captializing `M` seems to be correct. Could you please explain this change?

Reply:
Thank you for your response.

In Hungarian, all languages are spelt with a lowercase letter.

The `suomi` languages also spells languages with a lowercase letter.

Below is a screenshot of the langues selector at the bottom of the [Thunderbird webpage](https://www.thunderbird.net/).
![image](https://github.com/user-attachments/assets/7f099876-1559-4113-9863-c738225f7662)

Thank you once again.